doc/man/zcashd.1

```diff
@@ -51,6 +51,10 @@ Run in the background as a daemon and accept commands
 .IP
 Specify data directory
 .HP
+\fB\-paramsdir=\fR<dir>
+.IP
+Specify Zcash network parameters directory
+.HP
```

Comment:
Remove this change (we regenerate this file automatically during releases, which is why it now conflicts).

Reply:
removed